src/ssd.c

```diff
@@ -75,14 +75,20 @@ ssd_box(struct view *view, enum ssd_part_type type)
 		box.width = theme->title_height;
 		box.height = theme->title_height;
 		break;
+	case LAB_SSD_BUTTON_WINDOW_MENU:
+		box.x = view->x;
+		box.y = view->y - theme->title_height;
+		box.width = theme->title_height;
+		box.height = theme->title_height;
+		break;
 	case LAB_SSD_PART_TITLEBAR:
 		box.x = view->x;
 		box.y = view->y - theme->title_height;
 		box.width = view->w;
 		box.height = theme->title_height;
 		break;
 	case LAB_SSD_PART_TITLE:
-		box.x = view->x + corner_square / 2;
+		box.x = view->x + theme->title_height + corner_square / 2;
```

Comment:
Shouldn't the offset here by `corner_square` rather than `theme->title_height`. I think they'll pretty close most of the time, but I think we're allowing a width of `corner_square` per icon.

Reply:
The other buttons use `theme->title_height` for their `box.x` offsets and for `width` and `height`.
I was confused in the calculation of `box.width = view->w - NR_BUTTONS * corner_square` as well, because that one instead indeed uses `corner_square` for the size of a button.

I'd welcome a second look into this.

Reply:
Sorry. Yes, theme->title_height is probably better.

Reply:
See #221